### PR TITLE
.coafile: Add commit message checking (BUGGED)

### PR DESCRIPTION
**Pull request by Mixih**
When using coala to check the files before comitting,
check for proper formatting of the commit message

Closes https://github.com/coala/coala-html/issues/38

- [x] I have rebased properly. Please see this for a tutorial.
- [x] I have gone through the commit guidelines and I've followed them:
- [x] I have followed the guidelines for the commit shortlog.
- [x] I have followed the guidelines for the commit body.
- [x] I have included the issue URL.
- [x] I have used the Fixes keyword if the commit fixes a real bug and the Closes keyword if it adds a feature/enhancement. See more here.
- [x] I have run all the tests and they all pass. That includes that all CI (below) is green - if GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -28,6 +28,11 @@ bears = PyUnusedCodeBear
 default_actions =
     PyUnusedCodeBear: ApplyPatchAction
 
+[commit]
+bears = GitCommitBear
+shortlog_trailing_period = false
+shortlog_regex = ([^:]*: [A-Z].*)
```
Comment (adtac) [inline .coafile]: coala and coala-bears use [this](https://github.com/coala/coala/blob/master/.coafile#L64). Any reason for a different regex?

Comment (Mixih) [inline .coafile]: Nope, just my interpretation of the guidelines. Plus, [^:]* seems redundant with [^:]+ . Should I make mine conform?

**Comment by adtac** [comment]
ack https://github.com/coala/coala-html/pull/88/commits/f8a3bf89628f88796bd999c15118b82447aaf9a7

**Comment by adtac** [comment]
@rultor merg

**Comment by adtac** [comment]
@rultor merge

**Comment by sils** [comment]
@rultor merge

**Comment by jayvdb** [comment]
unack f8a3bf8

**Comment by jayvdb** [comment]
As already aware on gitter, the builds are green but all is not well : https://travis-ci.org/coala/coala-html/builds/177549877#L445

**Comment by Mixih** [comment]
After looking through the circle, I believe the circle warnings aren't a problem. Coala-html doesn't need the bears to work. bears are only needed during dev linting. We can fix this by adding something to the circle config (which would be separate issue). The warnings have been the ere the last few pushes too. See https://travis-ci.org/coala/coala-html/jobs/150564234

**Comment by tushar-rishav** [comment]
@Mixih I think we may not need 4490cf0a9dbce4640982 anymore as https://github.com/coala/coala-html/pull/92 is merged now.
Also, we may close https://github.com/coala/coala-html/pull/89 as https://github.com/coala/coala-html/pull/88 already has the changes from former one. So let's work on #88 only? Correct me if I miss something here. :)

**Comment by jayvdb** [comment]
> @Mixih I think we may not need 4490cf0 anymore as #92 is merged now.

4490cf0 is the same commit from #92 ; see the [proposed commits](https://github.com/coala/coala-html/pull/92/commits) and now the git log.

rultor should hopefully automatically resolve this and merge one new commit for this PR.

**Comment by jayvdb** [comment]
ack 34dfae3

**Comment by jayvdb** [comment]
@rultor merge

**Comment by tushar-rishav** [comment]
reack 85df54a437e001672f

**Comment by jayvdb** [comment]
ack 6684653

**Comment by jayvdb** [comment]
@rultor merge

**Comment by jayvdb** [comment]
This PR is an excellent setup for https://github.com/coala/coala-bears/issues/1037 .  At present, unless we fix GitCommitBear, this PR is unusable.

I suggest creating a new PR, and letting me keep playing with this PR and the @Mixih branch "Mixih/commit"

**Comment by jayvdb** [comment]
We have a few other test branches now, and it seems quite easy to create this scenario.

**Pull request closed without merging**